.github/workflows/cut-rc.yml

```
@@ -0,0 +1,547 @@
+                >> "$SLACK_MESSAGE_FILE"
+              fi
+            else
+              echo "Already picked cherries for commit $COMMIT:"
+              echo "$ALREADY_PICKED_CHERRIES"
+              echo "..."
+            fi
+          done
+
+          git push
+          if [ -n "$EMPTY" ]; then
+            echo "Cherry-picked commits have been pushed to the release branch." >> $GITHUB_STEP_SUMMARY
+          fi
+
+          # Replace % with %25 and join lines with %0A:
+          FAILED="$(sed ':a; N; $!ba; s/%/%25/g; s/\n/%0A/g' $SLACK_MESSAGE_FILE)"
+          rm -f "$SLACK_MESSAGE_FILE"
+
+          if [ -n "$FAILED" ]; then
+            echo "::set-output name=bad-cherries::$FAILED"
+            cat << EOF >> $GITHUB_STEP_SUMMARY
+
+          As some of the PRs could not be cherry-picked, please help the authors to merge their changes to the release branch via opening PRs.
+          The commands may look like the following:
+
+              git pull
+              git switch "${{needs.variables.outputs.branch}}"
+              git switch --create <AUTHOR>/merge-pr-<PR NUMBER>-"${{needs.variables.outputs.release}}"
+              git cherry-pick -x <MERGE COMMIT SHA>
+              # Proceed to resolve merge conflicts. Once done:
+              git push --set-upstream origin $(git branch --show-current)
+
+              # Create PR to the release branch:
+              gh create pr --base "${{needs.variables.outputs.branch}}" --fill \\
+                --milestone "${{needs.variables.outputs.milestone}}"
+
+          EOF
+            exit 1
+          fi
+      - name: Post to Slack
+        if: always() && steps.cherry-pick.outputs.bad-cherries != ''
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+        uses: slackapi/slack-github-action@v1.19.0
+        with:
+          channel-id: ${{env.slack_channel}}
+          payload: |
+            { "blocks": [ {
+              "type": "header", "text": { "type": "plain_text",
+                "text": "Couldn't cherry-pick the following PRs to the release branch:"
+              }},{
+              "type": "section", "text": { "type": "mrkdwn",
+                "text": "${{steps.cherry-pick.outputs.bad-cherries}}"
+            }}]}
+
+  cut-rc:
+    name: Tag RC for milestone ${{needs.variables.outputs.milestone}}
+    runs-on: ubuntu-latest
+    needs: [variables, check-merged-prs, check-jira]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: "${{needs.variables.outputs.branch}}"
+      - name: Initialize mandatory git config
+        run: |
+          git config user.name "${{github.event.sender.login}}"
+          git config user.email noreply@github.com
+      - name: Update docs submodule
+        run: |
+          git submodule update --init --remote -- docs/content
+          git add docs/content
+          if ! git diff-index --quiet HEAD; then
+            git commit -m "Update docs submodule"
```

```suggestion
            git commit -m "Update docs submodule for milestone ${{needs.variables.outputs.milestone}}"
```
? (no idea whether that would actually work)